From dc6e6d7a2bedcc34d59e76f08ca4362369438f32 Mon Sep 17 00:00:00 2001
From: David Heidelberg <david.heidelberg@collabora.com>
Date: Tue, 28 May 2024 14:15:09 -0700
Subject: [PATCH] freedreno: Enable Adreno 306A

Enable the Adreno 306A that is found on the QM215 SoC (Qualcomm 215).

The GPU marketing name is Adreno 308.

Kernel patch: https://lore.kernel.org/linux-arm-msm/20240528-a306a-v1-1-03a66dacd8c7@gmail.com/

Signed-off-by: David Heidelberg <david.heidelberg@collabora.com>
Part-of: <https://gitlab.freedesktop.org/mesa/mesa/-/merge_requests/29401>
---
 src/freedreno/common/freedreno_devices.py | 1 +
 1 file changed, 1 insertion(+)

diff --git a/src/freedreno/common/freedreno_devices.py b/src/freedreno/common/freedreno_devices.py
index 2d38d01862b7f..b3ad9fbc4b9f0 100644
--- a/src/freedreno/common/freedreno_devices.py
+++ b/src/freedreno/common/freedreno_devices.py
@@ -217,6 +217,7 @@ add_gpus([
         GPUId(320),
         GPUId(330),
         GPUId(chip_id=0x03000512, name="FD305B"),
+        GPUId(chip_id=0x03000620, name="FD306A"),
     ], GPUInfo(
         CHIP.A3XX,
         gmem_align_w = 32,  gmem_align_h = 32,
-- 
GitLab

